core-services/release-controller/_releases/release-ocp-4.17-multi.json

```diff
@@ -11,18 +11,24 @@
   "check": {},
   "publish": {},
   "verify": {
+    "multi-e2e-ovn": {
```

Comment:
Can we update the short name as well, I think we talked about anything after the <release> in the periodic name.
in this case (assuming we can drop 'ocp'):  `e2e-aws-ovn-multi-x-xa`

Reply:
Yes, I can lengthen the short name.  Will update.